src/ripple/test/jtx/JTx.h

```diff
@@ -56,12 +59,99 @@ struct JTx
     {
     }
 
+#if defined(_MSC_VER) && _MSC_VER <= 1800
```

Comment:
Not in this changeset, but the two constructors on lines 52 and 57 can be simplified by taking the param by value and moving: `JTx (Json::Value jv_):jv(std::move(jv_){}`. These are also not explicit, should they be?